### PR TITLE
Add property to set tabs on bottom

### LINKED ISSUE #4214: feature TabContainer option: tab position top/bottom
**Issue description** (what happened, and what was expected):
I cont create a tab container where the tabs are listed at the bottom so I have to programm one by myself. (HBottonarray and a script...)

**missing feature**
there should be an option in the tabContainer Node that lets me choose if I want to have the tab at the bottom or at the top. (Like the option tab align: left/right/center or: tab visible true/false)

### PR DESCRIPTION
**Pull request by tomasantunes**
Add property to set tabs on bottom. Fixes #4214

### CODE REVIEW

File: scene/gui/tabs.cpp [hunk 1]
```diff
@@ -340,21 +347,36 @@ void Tabs::_notification(int p_what) {
...
 				Size2i sb_ms = sb->get_minimum_size();
 				Ref<Texture> icon = tabs[i].icon;
+
 				if (icon.is_valid()) {
 
-					icon->draw(ci, Point2i(w, sb->get_margin(MARGIN_TOP) + ((sb_rect.size.y - sb_ms.y) - icon->get_height()) / 2));
+					if (position == BOTTOM) {
+						icon_y = get_size().height - icon->get_height() / 2;
+					}
+					else {
```
Comment (akien-mga) [inline scene/gui/tabs.cpp]: `else {` should be on the same line as the previous `}` as per our style guide, see https://travis-ci.org/godotengine/godot/jobs/417610905 and http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

**Comment by LikeLakers2** [comment]
Noticed AppVeyor is still marked as pending despite the AppVeyor build being canceled. @akien-mga Not sure if you know about this or not.

**Comment by Calinou** [comment]
@tomasantunes Could you look at resolving conflicts and squashing commits into a single one? :slightly_frowning_face:

**Comment by Zireael07** [comment]
I don't think it worked, you still seem to have conflicting files?

**Comment by Calinou** [comment]
@tomasantunes There are many unrelated changes from upstream commits still included in this pull request, could you take a look at solving this?

PS: You can use `git reflog` to go back to a specific state in your local copy of the repository (to an extent), even if you amended and force-pushed a commit.

**Comment by akien-mga** [comment]
Something went wrong again, this PR changes 5033 files with +763,239 −392,389 lines of code...

I close it as it's not in a mergeable state and is likely beyond fixing now. Do not hesitate to open a new PR reimplementing this feature on top of the current master branch. Also see [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html) for tips on Git usage for the workflow we use.

**Pull request closed without merging**